src/main/java/edu/rpi/legup/puzzle/skyscrapers/SkyscrapersType.java

```diff
@@ -8,4 +8,31 @@ public enum SkyscrapersType {
     SkyscrapersType(int value) {
         this.value = value;
     }
+
+    public static SkyscrapersType convertToSkyType(int num) {
```

Comment:
This can be simplified to a `switch` statement instead of multiple `if else` statements.